src/components/ChatLog.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+import './ChatLog.css'
+import PropTypes from 'prop-types';
+
+const ChatLog = (props) => {
+    const chatLog = 'chat-log';
+    let showLike;
+    console.log(props.entries);
```

Comment:
👀 Remove debugging logs from projects. This makes a lot of console noise when running tests (and prints out to the console in the browser).